webpack.config.js

```diff
@@ -64,7 +53,7 @@ const plugins = [
     },
   }),
   new HtmlWebpackPlugin({
-    title,
+    title: configFile.components.appName,
```

Comment:
I think we need a default value for this (before it would be `Generic Data Commons`)?